packages/e2e-tests/tests/stepping-05_chromium.test.ts

```diff
@@ -33,19 +33,14 @@ test(`stepping-05_chromium: Test stepping in pretty-printed code`, async ({
   await resumeToLine(page, 8);
   await stepOverToLine(page, 8);
 
-  // TODO Stepping here hangs
-  // await stepOverToLine(page, 9);
+  // TODO: extra test needed.
+  await stepOverToLine(page, 8);
+  await stepOverToLine(page, 9);
```

Comment:
What does that TODO comment mean?

Can you add more explanation and/or link to a Linear task?